### PR TITLE
[ZEPPELIN-4764]. Support real time dashboard for jdbc interpreter

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
This PR is to support build real time dashboard via jdbc interpreter.  User just need to specify refreshInterval to enable it.  e.g.

```
%mysql(refreshInterval=2000)

select * from flink_cdn.cdn_access_statistic;
```

### What type of PR is it?
[Feature ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4764

### How should this be tested?
* Unit test is added, also manually tested.

### Screenshots (if appropriate)
jdbc_time

jdbc_refresh

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: jdbc/pom.xml [hunk 1]
```diff
@@ -33,6 +33,144 @@
   <version>0.9.0-SNAPSHOT</version>
   <name>Zeppelin: JDBC interpreter</name>
 
+  <properties>
+    <!--library versions-->
+    <interpreter.name>jdbc</interpreter.name>
+    <postgresql.version>9.4-1201-jdbc41</postgresql.version>
+    <hadoop.common.version>2.7.2</hadoop.common.version>
+    <h2.version>1.4.190</h2.version>
+    <commons.dbcp2.version>2.0.1</commons.dbcp2.version>
```
Comment (alexott) [inline jdbc/pom.xml:42]: why do we use version that is released in 2014th? Maybe it makes sense to upgrade to some later version?

File: jdbc/pom.xml [hunk 2]
```diff
@@ -33,6 +33,144 @@
...
+    <!--test library versions-->
+    <mockrunner.jdbc.version>1.0.8</mockrunner.jdbc.version>
+  </properties>
+
+  <dependencies>
+
+	<dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${postgresql.version}</version>
```
Comment (alexott) [inline jdbc/pom.xml:53]: should it have `test` scope as well? Why we need specific JDBC artifact in Zeppelin itself?

File: jdbc/pom.xml (hunk 1 above)
Comment (zjffdu) [reply jdbc/pom.xml:42]: Yes, agree.

File: jdbc/pom.xml (hunk 2 above)
Comment (zjffdu) [reply jdbc/pom.xml:53]: I believe it is because we use postgresql by default in jdbc interpreter. This is by design I think.

**Comment by zjffdu** [comment]
@alexott Thanks for your comment. The issues you mentioned make sense to me. But I think it is not related to this ticket. We could do it in another ticket. I do think we need to refactor and revisit the design of jdbc interpreter.

**Comment by alexott** [comment]
@zjffdu I agree with that - that's why I commented only :-)
Otherwise, LGTM for me

**Pull request closed without merging**